docs/scripting/functions/format.md

```diff
@@ -21,6 +21,24 @@ Formats a string to include variables and other strings inside it.
 
 This function does not return any specific values.
 
+## Format Specifiers
+
+| Specifier | Meaning                                       |
+| --------- | --------------------------------------------- |
+| %i        | Unsigned Integer                              |
+| %d        | Signed Integer                                |
+| %s        | String                                        |
+| %f        | Floating-point number                         |
+| %c        | ASCII character                               |
+| %x        | Hexadecimal number                            |
```

Comment:
It should be noted that this specifier is broken, or does not behave as expected. I forgot exactly how it's broken though, someone on the discord should be able to answer that.